ct/listmonk.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: bvdberg01
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+function header_info {
+clear
+cat <<"EOF"
+    ___      __                        __  
+   / (_)____/ /_____ ___  ____  ____  / /__
+  / / / ___/ __/ __ `__ \/ __ \/ __ \/ //_/
+ / / (__  ) /_/ / / / / / /_/ / / / / ,<   
+/_/_/____/\__/_/ /_/ /_/\____/_/ /_/_/|_|  
+                                           
+EOF
+}
+header_info
+echo -e "Loading..."
+APP="listmonk"
+var_disk="4"
+var_cpu="1"
+var_ram="512"
```

Comment:
Is this really enough ?

Reply:
![{76A3EAE6-78CF-4135-9D9F-5C37BEF2327D}](https://github.com/user-attachments/assets/37db2540-9114-4b21-859d-849a63b38d46)
This is what listmonk has on there website, so thats my I thought it was enough.